code/html/index.html

```diff
@@ -297,7 +297,7 @@ <h2>General configuration values</h2>
                                     <div class="pure-u-0 pure-u-lg-1-4"></div>
                                     <div class="pure-u-1 pure-u-lg-3-4 hint">
                                         This setting defines the behaviour of the main LED in the board.<br />
-                                        When in "WiFi status" it will blink at 1Hz when trying to connecting. If successfully connected if will briefly lit every 5 seconds if in STA mode or every second if in AP mode.<br />
+                                        When in "WiFi status" it will blink at 1Hz when trying to connect. If successfully connected if will briefly blink every 5 seconds if in STA mode or every second if in AP mode.<br />
```

Comment:
While you're fixing this, I'd recommend changing `If successfully connected if will briefly blink every...` to `If successfully connected it will briefly blink every...`

Reply:
Done